server/accounts.go

```diff
@@ -611,6 +611,18 @@ func (m1 *ServiceLatency) merge(m2 *ServiceLatency) {
 	m1.NATSLatency.System = m1.ServiceLatency - (m2.ServiceLatency + m2.NATSLatency.Responder)
 	m1.ServiceLatency = m2.ServiceLatency
 	m1.NATSLatency.Responder = m2.NATSLatency.Responder
+
+	// Adjust to zero in case previous calculations resulted in negative values.
+	sanitizeLatencyMetric(m1)
+}
+
```

Comment:
Add comment about why.

Since we estimate client RTT for both requestor and responder these numbers could go negative in some edge conditions. These numbers are never meant to be negative, it just could be how we back into the values based on estimated RTT.

Reply:
sgtm, updated with comment based on above.